loader/src/main/java/net/neoforged/fml/config/IConfigSpec.java

```diff
@@ -53,5 +52,20 @@ public interface IConfigSpec {
      * This is called on loading and on reloading.
      * The config is guaranteed to be valid according to {@link #isCorrect}.
      */
-    void acceptConfig(@Nullable CommentedConfig config);
+    void acceptConfig(@Nullable ILoadedConfig config);
+
+    sealed interface ILoadedConfig permits LoadedConfig {
```

Comment:
This is a sealed interface, so is there any reason to have it over a normal class?

Reply:
Just hiding implementation details. :wink: